gym/wrappers/record_episode_statistics.py

```diff
@@ -9,33 +10,34 @@
 class RecordEpisodeStatistics(gym.Wrapper):
     """This wrapper will keep track of cumulative rewards and episode lengths.
 
-    At the end of an episode, the statistics of the episode will be added to `info`. After the completion
-    of an episode, `info` will look like this:
+    Example:
+        At the end of an episode, the statistics of the episode will be added to `info`. After the completion
+        of an episode, `info` will look like this:
 
```

Comment:
Proper code highlighting
```suggestion
        At the end of an episode, the statistics of the episode will be added to ``info``. After the completion
        of an episode, ``info`` will look like this::

```